### PR TITLE
Ability to translate a feature. (with ModeHandler)

### PR DESCRIPTION
**Pull request by sivako**
This should go after #101 

Adding the new "translate" mode on top of ModeHandler refactor
nebula-translate-mode-v2
.
Scenario: Select mode as "translate" -> Select any feature -> mouse click + hold on feature and drag to move to desired location.

### CODE REVIEW

**Comment by supersonicclay** [comment]
@sivako please rebase on master and open a new PR against master

**Comment by georgios-uber** [comment]
@sivako please fix conflicts

File: examples/deck/example.js [hunk 1]
```diff
@@ -197,6 +197,7 @@ export default class Example extends Component<
             >
               <option value="view">view</option>
               <option value="modify">modify</option>
+              <option value="translate">translate</option>
```
Comment (supersonicclay) [inline examples/deck/example.js]: nit: also, ignore `translating` when doing logging from `onEdit`.

File: modules/core/src/lib/mode-handlers/translate-handler.js [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+import type {
+  PointerMoveEvent,
+  StartDraggingEvent,
+  StopDraggingEvent,
+  DeckGLPick
+} from '../event-types.js';
+import type { EditAction } from './mode-handler.js';
+import { ModeHandler } from './mode-handler.js';
+
+export class TranslateHandler extends ModeHandler {
```
Comment (supersonicclay) [inline modules/core/src/lib/mode-handlers/translate-handler.js]: nit: i see this is based on an old version of `RotateHandler`. Might be cleaner if `RotateHandler`, `ScaleHandler`, and `TranslateHandler` are all more closely aligned in their logic.

File: modules/core/src/lib/mode-handlers/translate-handler.js [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+      selectedFeatureIndexes.length === 1
+        ? this.getFeatureCollection().features[selectedFeatureIndexes[0]]
+        : null;
+
+    return picks.some(p => p.object === singleSelectedFeature);
+  }
+
+  getCursor({ isDragging }: { isDragging: boolean }): string {
+    if (this._isTranslatable) {
+      // TODO: look at doing SVG cursors to get a better "translate" cursor
```
Comment (supersonicclay) [inline modules/core/src/lib/mode-handlers/translate-handler.js]: TODO doesn't apply. `move` probably the perfect cursor for translate mode.

**Comment by sivako** [comment]
Thank you @georgios-uber and @supersonicclay.  Addressed the comments as part of #109

**Pull request merged**